pkg/abstractions/image/image.go

```diff
@@ -149,6 +169,21 @@ func (is *RuncImageService) BuildImage(in *pb.BuildImageRequest, stream pb.Image
 	return nil
 }
 
+func (is *RuncImageService) parseBuildSecrets(ctx context.Context, secrets []string, authInfo *auth.AuthInfo) ([]string, error) {
```

Comment:
nitpick but can this be renamed to `retrieveBuildSecrets`?

Reply:
done!